api/src/main/java/io/strimzi/api/kafka/model/kafka/quotas/QuotasPluginStrimzi.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.api.kafka.model.kafka.quotas;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import io.strimzi.api.kafka.model.common.Constants;
+import io.strimzi.crdgenerator.annotations.Description;
+import io.strimzi.crdgenerator.annotations.Maximum;
+import io.strimzi.crdgenerator.annotations.Minimum;
+import io.sundr.builder.annotations.Buildable;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+
+import java.util.List;
+
+@Buildable(
+    editableEnabled = false,
+    builderPackage = Constants.FABRIC8_KUBERNETES_API
+)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonPropertyOrder({"type", "producerByteRate", "consumerByteRate", "minAvailableBytesPerVolume", "minAvailableRatioPerVolume", "excludedPrincipals"})
+@EqualsAndHashCode(callSuper = true)
+@ToString(callSuper = true)
+public class QuotasPluginStrimzi extends QuotasPlugin {
+    private static final long serialVersionUID = 1L;
+
+    public static final String TYPE_STRIMZI = "strimzi";
+
+    private Long producerByteRate;
+    private Long consumerByteRate;
+    private Long minAvailableBytesPerVolume;
+    private Double minAvailableRatioPerVolume;
+    private List<String> excludedPrincipals;
+
+    @Description("Must be `" + TYPE_STRIMZI + "`")
+    @Override
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    public String getType() {
+        return TYPE_STRIMZI;
+    }
+
+    @Description("A quota on the maximum bytes per-second that each client group can publish to a broker before " +
+        "the clients in the group are throttled. Defined on a per-broker basis.")
```

Comment:
Yeah this description is wrong, it should be byte-rate for all of the clients per cluster (I think).
So something like 

```
A total quota limiting byte-rate for clients, independent on their number, that produce to a cluster, defined on a per-cluster basis. The limitation depends on the clients produce speed - in case that clients are producing messages as fast as possible, the limitation will be equal for all. In other case when one is producing at specific rate, the other one is limited by the remainder of the difference between the specified byte-rate and the speed of the first.
```
but I guess it's too long for a description 😄 

Same applies for the fetch quota

WDYT @scholzj ?